Tabs activity

balthazzar opened the pull request:
fixed tabs activity
fixed titter comments alert

balthazzar committed:
Tabs fix

balthazzar committed:
Test fix

AlexeyAnshakov commented on WRIO-InternetOS/js/components/CreateDomCenter.js:
```diff
@@ -255,15 +256,13 @@ class CreateDomCenter extends React.Component {
-                condition = false;
-                this.state.nocomments = true;
+                condition = true;
                 displayWebgold = (<iframe src={'//webgold.'+process.env.DOMAIN+'/add_funds'} style={ this.editIframeStyles }/>);
```
@michbil should we use strict https?

michbil replied:
@AlexeyAnshakov @balthazzar definetly, we should switch to https there

balthazzar committed:
Alert fix

balthazzar committed:
Titter widget display

allow titter for type Article

michbil commented on WRIO-InternetOS/js/components/CreateDomCenter.js:
```diff
@@ -182,7 +183,7 @@ class CreateDomCenter extends React.Component {
             notDisplayCenter: false,
             byButton: true,
-            nocomments: false
+            condition: false
```
@balthazzar Думаю можно пойти еще дальше и попробовать дать этой переменной "говорящее" имя, чтобы было понятно что она обозначает с первого раза. Мое предложение - displayTitterCondition. Возможно у тебя будет лучший вариант.

balthazzar committed:
Tabs state save

tabs state saving add
change condition to displayTitterCondition
getting last order fixed

balthazzar committed:
https fix
